cmd/kubeadm/app/cmd/init.go

```diff
@@ -136,7 +136,17 @@ func RunInit(out io.Writer, cmd *cobra.Command, args []string, s *kubeadmapi.Kub
 
 	// TODO(phase1+) create a custom flag
 	if s.InitFlags.CloudProvider != "" {
-		if _, ok := kubeadmapi.SupportedCloudProviders[s.InitFlags.CloudProvider]; !ok {
+		found := false
+		for _, provider := range kubeadmapi.ListOfCloudProviders {
+			if provider == s.InitFlags.CloudProvider {
+				found = true
+				break
+			}
+		}
+
+		if found {
+			fmt.Printf("<cmd/init> cloud provider %q initialized for the control plane. Remember to set the same cloud provider flag on the kubelet.", s.InitFlags.CloudProvider)
```

Comment:
Missing `\n` at the end of this Printf statement. Have you tested this manually and found that the code works?